src/Aspire.Dashboard/Components/ResourcesGridColumns/SourceColumnDisplay.razor

```diff
@@ -3,14 +3,14 @@
 
 @if (Resource is ProjectViewModel projectViewModel)
 {
-    <span title="@projectViewModel.ProjectPath" aria-label="@projectViewModel.ProjectPath">@projectViewModel.ProjectPath.TrimMiddle(50)</span>
+    <span title="@projectViewModel.ProjectPath" aria-label="@projectViewModel.ProjectPath">@Path.GetFileName(projectViewModel.ProjectPath)</span>
```

Comment:
Also remove TrimMiddle method if not used anymore?

Reply:
It's still used for the exe working directory.